pool/rate_limiting_average.go

```diff
@@ -86,7 +73,7 @@ func (p *ewmaRateLimiting) getNextTime(dur time.Duration) time.Duration {
 	tasksPerPeriod := p.period / averageRuntime
 
 	// the capacity of the pool is the size of the pool and the
```

Comment:
`The`

Reply:
Fixed this comment and the other comments in `getNextTime`.